src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import './App.css'
+import Message from './components/greeting'
+
+function App() {
+
+  return (
+    <>
+      <Message />
+    </>
+  )
```

Comment:
- [ ] As pointed out by the last reviewer, kindly set a route to the `message` component using `React-Router` as instructed in the project requirement 👍 

### Instruction:
![image](https://github.com/danielochuba/hello-react-front-end/assets/50990925/f6b43051-38f5-431f-af7c-cd42fc00db05)